src/_modules/experimental/Request.sol

```diff
@@ -34,26 +36,46 @@ struct RequestBuilder {
 struct Request {
     Method method;
     string url;
-    Header[] headers;
+    RequestHeaders headers;
     bytes body;
 }
 
 type RequestResult is bytes32;
 
-// TODO: headers, etc
+type RequestHeaders is bytes32;
+
 struct Response {
     string url;
     uint256 status;
+    ResponseHeaders headers;
     bytes body;
 }
 
 type ResponseResult is bytes32;
 
+type ResponseHeaders is bytes32;
+
 library request {
     using request for *;
+    using LibHeaders for *;
 
     // Return an empty client
-    function create() internal pure returns (RequestClient memory) {}
+    function create() internal returns (RequestClient memory client) {
+        Command memory curlVersionCmd = commands.create("bash").args(
+            [
+                "-c",
+                "version=$(curl --version | grep -oE 'curl [0-9]+\\.[0-9]+\\.[0-9]+' | awk '{print $2}');echo $version"
+            ]
+        );
+
+        CommandOutput memory curlVersion = curlVersionCmd.run().unwrap();
```

Comment:
should we have a better way to handle if there is an error here? maybe use `expect(...)` instead of unwrap